tensorflow/contrib/tensorrt/BUILD

```diff
@@ -88,6 +88,7 @@ cc_library(
         ":trt_logging",
         ":trt_resources",
         "//tensorflow/core:gpu_headers_lib",
+        "//tensorflow/core:gpu_runtime",
```

Comment:
Please help to sync to tensorflow:master, and I'll import this PR for internal test after that.